### PR TITLE
Add spinners to reset teammember row when button is pressed

### PR DESCRIPTION
**Pull request by buoyad**
This adds a spinner to the button that was clicked and disables the other one. r? @keybase/react-hackers

### CODE REVIEW

File: shared/actions/json/teams.json [hunk 1]
```diff
@@ -64,7 +64,6 @@
     },
     "addToTeam": {
       "teamname": "string",
-      "email": "string",
```
Comment (buoyad) [inline shared/actions/json/teams.json]: This wasn't used by anything.

File: shared/teams/team/members/member-row/index.js [hunk 2]
```diff
@@ -126,8 +128,22 @@ export const TeamMemberRow = (props: Props) => {
           props.youCanManageMembers && (
             <Box style={{...globalStyles.flexBoxRow, flexShrink: 1}}>
               <ButtonBar>
-                <Button small={true} label="Re-Admit" onClick={props.onReAddToTeam} type="PrimaryGreen" />
-                <Button small={true} label="Remove" onClick={props.onRemoveFromTeam} type="Secondary" />
+                <Button
```
Comment (buoyad) [inline shared/teams/team/members/member-row/index.js]: just added `waiting` and `disabled` to all of these

File: shared/actions/teams.js [hunk 3]
```diff
@@ -263,11 +264,16 @@ const _removeMemberOrPendingInvite = function*(action: TeamsGen.RemoveMemberOrPe
     throw new Error(errMsg)
   }
 
-  yield Saga.put(createIncrementWaiting({key: Constants.teamWaitingKey(teamname)}))
+  // only one of (username, email, inviteID) is truth-y
+  const waitingKeys = [
+    Constants.teamWaitingKey(teamname),
+    Constants.removeMemberWaitingKey(teamname, username || email || inviteID),
```
Comment (akalin-keybase) [inline shared/actions/teams.js]: use `invitees` from above?

Comment (buoyad) [reply shared/actions/teams.js]: yup, thanks!

**Pull request merged**